benchmarks/common.py

```diff
@@ -792,6 +792,71 @@ def decay_batch_exp(self, batch_size, factor=0.5, divisor=2):
             out_batch_size = batch_size - 1
         return max(0, int(out_batch_size))
 
+    def measure_latency_and_memory(self, device, model_name, model_iter_fn, backends):
+        get_peak_memory = lambda: torch.cuda.max_memory_allocated() / 10**9
```

Comment:
Device is passed into `measure_latency_and_memory` but not `max_memory_allocated` which could lead to confusion. Is this intentional?

Reply:
Nah, I am just bad at naming :)

Will try to name things better.